src/rsk/rskEngine.js

```diff
@@ -160,7 +160,10 @@ export class RskEngine extends CurrencyEngine {
         this.updateBalance('RBTC', balance)
       }
     } catch (e) {
-      this.log(`Error checking token balance: RBTC`)
+      this.log(`Error checking token balance: RBTC`, e)
```

Comment:
This needs to actually detect for the 404 "balance not found" before it calls update balance. Otherwise if there is a network error then we will set the balance to zero

Reply:
Yes, I totally understand (I tried to get the message but I couldn't. Let's try harder).